web/auth/auth.go

```diff
@@ -614,6 +614,7 @@ func Routes(router *echo.Group) {
 
 	// Flagship app
 	router.POST("/session_code", CreateSessionCode)
+	router.POST("/:slug/access_token", impersonatedAccessToken)
```

Comment:
I would prefer to avoid the `access_token` naming. `access_token` is generally used for OAuth clients, and a konnector doesn't have an OAuth client. And I would like to have `konnectors` somewhere in the route. Maybe `POST /auth/tokens/konnectors/:slug`.